src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.junit.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@TestPropertySource(
+    properties = {
+        "feature_toggles.emailToStaff=true"
```

Comment:
as it is defaulted to true, no need to set this property, instead test for false value, confirming staff email is not send.

Reply:
I'd actually leave it this way to make it clear that we want top test different config value than in other tests.

What I'd suggest to add is Java doc above that would describe what is this test for.